app/build.gradle

```diff
@@ -43,6 +43,8 @@ dependencies {
     implementation Dependencies.kotlin_stdlib
     implementation Dependencies.design
     implementation Dependencies.constraint_layout
+    implementation 'androidx.appcompat:appcompat:1.0.0-beta01'
```

Comment:
I suggested a fix on the `Dependencies.kt` file, once you do that you'l be able to put here `Dependencies.appcompat`